src/Instrumentation/OpenAIPHP/composer.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "open-telemetry/opentelemetry-auto-openai-php",
+  "description": "OpenTelemetry auto-instrumentation for openai-php/client",
+  "keywords": ["opentelemetry", "otel", "open-telemetry", "tracing", "io", "instrumentation", "openai"],
+  "type": "library",
+  "homepage": "https://opentelemetry.io/docs/php",
+  "readme": "./README.md",
+  "license": "Apache-2.0",
+  "minimum-stability": "dev",
+  "require": {
+    "php": "^8.2",
```

Comment:
why 8.2? the client claims to work with 8.1

Reply:
Lowered the requirement :)